### ISSUE TITLE
storing more than sound effect generated

### ISSUE THREAD

**Issue by BlueSteelAUS** [tomeyro/godot-sfxr#11]

**Comment by BlueSteelAUS** [comment]
sorry i can;t find how to delete this issue

**Comment by tomeyro** [comment]
@BlueSteelAUS was it a mistake or did you find a solution for what you were looking for? If the latter, it would be nice to keep the problem here and the solution in case someone looks for it as well.

**Comment by BlueSteelAUS** [comment]
I found the solution in the docs on Git Hub.  The sound resource file I didn't make it to put into where the waveform is. so I made it manually. and dragged it over .. and it worked. I had spent 6 hours trying to figure it out and another hour after I got here. then after oi made this issue. it finally made sense to me I was kicking myself big time it was the picture that finally triggered the solution. and made a resource of it and dragged the resource into where the waveform is displayed and it worked :) it looks a little ugly with the generator thins in there too but then that makes sense that hopefully(I haven't tried) to change the sound effect within the area where iv dragged the resource. I hope what I've said makes sense I'm only a beginner. I also had asked on Godot's Discord in the beginner area. and no one could tell me how to assign 2 different sounds to 2 buttons.. so I worked it all out myself finally, it's really simple but took me about 6-7 hours to figure it out. BTW: I'd like to thank you very very much for making this ADDON/MODULE/whatever they call it. I've been looking for a programming language to add a version of SFXR to it so I can now have inbuilt sound data/etc. so I don't need to have and distribute .wav sound files. Thank you very very much. you have no idea how long I've waited for this feature.
